src/Ledger/Conway/Conformance/Gov.agda

```diff
@@ -46,61 +35,7 @@ instance
   unquoteDecl To-GovEnv = derive-To
     [ (quote GovEnv , To-GovEnv) ]
 
-private variable
-  Γ : GovEnv
-  s s' : GovState
-  aid : GovActionID
-  voter : Voter
-  vote : GovVote
-  v : Vote
-  d : Coin
-  addr : RwdAddr
-  a : GovAction
-  prev : NeedsHash a
-  k : ℕ
-  p : Maybe ScriptHash
-
-open GState
-open PState
-
-opaque
-  isRegistered : GovEnv → Voter → Type
-  isRegistered Γ (r , c) =
-    let open GovEnv Γ; open CertState certState in
-    case r of λ where
-      CC    → just c ∈ range (gState .ccHotKeys)
-      DRep  → c ∈ dom (gState .dreps)
-      SPO   → c ∈ mapˢ KeyHashObj (dom (pState .pools))
-
-data _⊢_⇀⦇_,GOV'⦈_  : GovEnv × ℕ → GovState → GovVote ⊎ GovProposal → GovState → Type where
-
-  GOV-Vote : ∀ {x ast} → let
-      open GovEnv Γ
-      vote = record { gid = aid ; voter = voter ; vote = v ; anchor = x }
-    in
-    ∙ (aid , ast) ∈ fromList s
-    ∙ canVote pparams (action ast) (proj₁ voter)
-    ∙ isRegistered Γ voter
-    ∙ ¬ (expired epoch ast)
-      ───────────────────────────────────────
-      (Γ , k) ⊢ s ⇀⦇ inj₁ vote ,GOV'⦈ L.addVote s aid voter v
-
-  GOV-Propose : ∀ {x} → let
-      open GovEnv Γ; open PParams pparams hiding (a)
-      prop = record { returnAddr = addr ; action = a ; anchor = x
-                    ; policy = p ; deposit = d ; prevAction = prev }
-      s' = L.addAction s (govActionLifetime +ᵉ epoch) (txid , k) addr a prev
-    in
-    -- ∙ actionWellFormed a
-    ∙ L.actionWellFormed a
-    ∙ L.actionValid rewardCreds p ppolicy epoch a
-    ∙ d ≡ govActionDeposit
-    ∙ L.validHFAction prop s enactState
-    ∙ L.hasParent enactState s a prev
-    ∙ addr .RwdAddr.net ≡ NetworkId
-    ∙ addr .RwdAddr.stake ∈ rewardCreds
-      ───────────────────────────────────────
-      (Γ , k) ⊢ s ⇀⦇ inj₂ prop ,GOV'⦈ s'
-
-_⊢_⇀⦇_,GOV⦈_   : GovEnv → GovState → List (GovVote ⊎ GovProposal) → GovState → Type
-_⊢_⇀⦇_,GOV⦈_ = ReflexiveTransitiveClosureᵢ {sts = _⊢_⇀⦇_,GOV'⦈_}
+_⊢_⇀⦇_,GOV⦈_ : GovEnv → GovState → List (GovVote ⊎ GovProposal) → GovState → Type
+Γ ⊢ govSt ⇀⦇ gvps ,GOV⦈ govSt'
+  = ⟦ txid , epoch , pparams , ppolicy , enactState , conv certState , rewardCreds ⟧ L.⊢ govSt |ᵒ conv certState ⇀⦇ gvps ,GOV⦈ govSt'
```

Comment:
What's the reason you moved `|ᵒ` here from `LEDGER`?

Reply:
If we called |ᵒ  in `Ledger` then we would have to expose the conversion function as well. In this way, I think is more modular.